Xamarin.Essentials/Calendars/Calendars.ios.cs

```diff
@@ -242,7 +243,7 @@ static RecurrenceRule GetRecurrenceRule(this EKRecurrenceRule iOSRule, NSTimeZon
 
         static RecurrenceRule ConvertToDayOfTheWeekList(this EKRecurrenceDayOfWeek[] recurrenceDays, RecurrenceRule rule)
         {
-            rule.DaysOfTheWeek = recurrenceDays.ToList().Select(x => (DayOfTheWeek)Convert.ToInt32(x.DayOfTheWeek)).ToList();
+            rule.DaysOfTheWeek = recurrenceDays.ToList().Select(x => (DayOfTheWeek)int.Parse(x.DayOfTheWeek.ToString(CultureInfo.InvariantCulture))).ToList();
```

Comment:
nints are a bit annoying, I'm not sure if there is a cleaner/nicer way to parse them? Or perhaps I should just use Convert.ToInt32 in this instance?

https://forums.xamarin.com/discussion/24048/xamarin-mac-64-bit-indexes-nint-cast-to-int

Reply:
can you not just force cast? ie (int)d.DayOfWeek? The point of the question on that link was more that he didn't feel the cast should be necessary, not that it didn't work.... but perhaps it doesn't

Reply:
Technically can do .Select(x => (DayOfTheWeek)(int)x.DayOfTheWeek)

Just not sure if I need to worry about nulls/bad cast (shouldn't happen but technically possible since nint is a int64)